nautobot/core/tests/test_graphql.py

```diff
@@ -1248,13 +1601,13 @@ def test_query_interfaces_connected_endpoint(self):
             name
             device { name }
         }
-        connected_console_server_port { id }
```

Comment:
Why remove this?

Reply:
I don't believe `ConsoleServerPort` is a valid connected endpoint for an interface.

https://github.com/nautobot/nautobot/blob/43b4f95eda43a17bf20557cc1e810b8fb386b786/nautobot/dcim/constants.py#L84